pandora-common/src/commands/executor.ts

```diff
@@ -80,7 +80,7 @@ export class CommandRunnerExecutor<
 
 	public autocomplete(): CommandAutocompleteResult {
 		return this.options.restArgName ? {
-			header: `🡆<${this.options.restArgName}>🡄`,
+			header: `⮕<${this.options.restArgName}>⬅`,
```

Comment:
I do not really like those new arrows, they are way too thin and look more like lines in the font size we use them in.

The purpose of these signs is to tell you at a glance the position in the command variable chain you are currently at if there are several ones.

Since the original one does not work on certain devices, I feel we should find something totally new to indicate the purpose.

How about `►<option>◄` ? Or something totally different and new~

Reply:
That were the closest one I could find in regards to what I was told it looked like.
<img width="260" alt="image" src="https://github.com/Project-Pandora-Game/pandora/assets/118102950/d0f24682-3768-40eb-8ca2-8315f3ce30cc">

The question would be if we need the arrows (or other symbols) in the first place?

Reply:
we could also try custom SVGs to make it work on any device and get the exact style we want

(I don't have a strong preference over `🡆`, `⮕`, `►`, to me all looks fine as long as it is better for most devices)

Reply:
> The question would be if we need the arrows (or other symbols) in the first place?

I mean nina specifically added that as a feature, so it maybe would be a shame to remove it as it could help someone?
Especially if we ever have more complex commands with 5 or more config values.